src/components/MyProfile/ProfileMain/ProfileMain.tsx

```diff
@@ -90,10 +92,7 @@ const ProfileMain = async () => {
               {profile?.seniority} {profile?.position} Developer
             </span>
             <div className={styles.addInfo}>
-              <div className={styles.addInfoItem}>
-                {' '}
-                {profile?.employmentType}
-              </div>
+            <div className={styles.addInfoItem}>{mapEmploymentType(profile?.employmentType)}</div>
```

Comment:
**nitpick:** Profile should not be optional, we should redirect to homepage if profile does not exist, or redirect to `create-profile` if user is authorized